dspace_metadata_import.sh

```diff
@@ -49,7 +85,7 @@ EOF
 
 echo "#####  Import metadata into the new database  #####"
 
-sudo -u postgres psql -U $DSPACE_DATABASE_OWNER -h localhost $NEW_DSPACE_DATABASE_NAME < $PATH_TO_OLD_DSPACE_DATABASE_DATA
+sudo -u postgres psql -U $DSPACE_DATABASE_OWNER -h $DATABASE_HOSTNAME $NEW_DSPACE_DATABASE_NAME < $PATH_TO_OLD_DSPACE_DATABASE_DATA
```

Comment:
I don't think this will handle authentication with a remote RDS database, which we'll need for prod.

Reply:
Need help from this. Do you have an example of this? Thanks.

Reply:
You can find some examples in https://github.com/OULibraries/dspace_vagrant
That work never made it to production, but the shell scripts are in pretty good shape.

The main thing is to switch to password-based authentication, which means that you'll probably want to set and use the `PGPASSWORD` environment variable in your scripts.